### PR TITLE
Update MapAction layers and POC report

### LINKED ISSUE #43: Update MapAction layers definition
Update the following layers:

- **mapaction_rail_length** : add type:relation
- **mapaction_lakes_count**/ **mapaction_lakes_area** : improve lake filter and add type:relation
- **mapaction_rivers_length** : add waterway:riverbank  and add type:relation

Add the following layer:

- **mapaction_major_roads_length** : more restrictive than existing major_roads_length length

### PR DESCRIPTION
**Pull request by ddebarros-mapaction**
Please add the **type of change** as label. If your PR is not ready for review and merge, please add `🚧 ` to the PR title.

### Description
The main change concerns the update of Map Action layers filters. We also changed the MapActionPoc report to be more consistent with this new layer definition.

### Corresponding issue
Closes #43 

### New or changed dependencies
-

### Checklist
- [x] I have updated my branch to `main` (e.g. through `git rebase main`)
- [X] My code follows the [style guide](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#style-guide) and was checked with [pre-commit](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CONTRIBUTING.md#pre-commit) before committing
- [X] I have commented my code
- ~I have added sufficient unit and integration [tests](https://github.com/GIScience/ohsome-quality-analyst/blob/main/docs/development_setup.md#tests)~
- [x] I have updated the [CHANGELOG.md](https://github.com/GIScience/ohsome-quality-analyst/blob/main/CHANGELOG.md)

Please check all finished tasks. If some tasks do not apply to your PR, please cross their text out (by using `~...~`) and remove their checkboxes.

### CODE REVIEW

**Comment by ddebarros-mapaction** [comment]
Not sure if I should add this modification to the CHANGELOG

**Comment by matthiasschaub** [comment]
@ddebarros-mapaction I have a general question about this PR: What is the reason to include the OSM type `relations` in the filter query?
I ask because this can lead to performance issue on the ohsome API side.

The share of objects of relation elements with the tag highway is very low (https://taginfo.openstreetmap.org/keys/highway). Also the OSM Wiki states that the element relation should not be used with the tag highway. The same is in part true for the other keys. Although there are often more objects of type relation.

On the syntax part, I would remove the whitespace around the operators such as the equal sign (https://docs.ohsome.org/ohsome-api/stable/filter.html).

You can include this in the CHANGELOG.md. Just put in the PR title and link to it.

Otherwise this looks good to me!

**Comment by ddebarros-mapaction** [comment]
Hello @matthiasschaub , thanks for the review. Just updated the two files (layer_definitions and CHANGELOG) with your recommendations.

Indeed, the type:relation is not indispensable. I first added it because some big lakes are represented with this type so I added it to be exhaustive. As this happened for the other layers as well I added everywhere in addition to type:way. 

In any case, we can live without it and it won't affect the quality report significantly.

**Review by matthiasschaub — APPROVED** [review]

**Review by matthiasschaub — DISMISSED** [review]

**Comment by ddebarros-mapaction** [comment]
Hello @matthiasschaub, just saw that there was a conflict concerning the CHANGELOG file. Let me know if it's OK now.

**Review by matthiasschaub — DISMISSED** [review]

**Review by matthiasschaub — APPROVED** [review]

**Comment by ddebarros-mapaction** [comment]
a new change on the main branch CHANGELOG was made. I had to resolve the conflict :/

**Review by matthiasschaub — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,14 @@
 - Add option to select different datasets and fid fields as input to OQT ([#4])
 - Rewrite save and load indicator from database logic to use one result table for all indicator results ([#37])
 - Remove GUF Comparison indicator ([#55])
-- Implement combine_indicators() as Concrete Method of the Base Blass Report ([53])
+- Implement combine_indicators() as Concrete Method of the Base Class Report ([53])
```
Comment (joker234) [inline CHANGELOG.md:11]:
```suggestion
- Implement combine_indicators() as Concrete Method of the Base Class Report ([#53])
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,12 +8,14 @@
...
 - Small changes to the Website ([#61])
   - Change HTML parameters from countryID and topic to id and report ([#30])
   - Mention API and GitHub on About page ([#5])
   - On Click to marker now only zooms to polygon instead to fixed value 
 - Return a GeoJSON when computing an indicator from the CLI using a dataset and FID ([#57])
+- Update MapAction layers and POC report ([#56])
+
```
Comment (joker234) [inline CHANGELOG.md:18]:
```suggestion
```

**Review by joker234 — CHANGES_REQUESTED** [review]

**Review by matthiasschaub — DISMISSED** [review]

**Review by matthiasschaub — APPROVED** [review]

**Review by matthiasschaub — APPROVED** [review]

**Review by matthiasschaub — APPROVED** [review]

**Review by matthiasschaub — APPROVED** [review]

**Comment by matthiasschaub** [comment]
@ddebarros-mapaction Yes this happens a lot to me as well with small changes after a review or merge with the main branch. I am going ahead an merge this branch now. Cheers :+1:

**Pull request merged**